spk/rsnapshot/src/requirements.txt

```diff
@@ -0,0 +1,2 @@
+# Cross compiled packages dependencies
```

Comment:
This file is only required when cross-compiling Python dependencies. From the PR, I gather that this does not apply ;)